src/infrastructure/content/ContentCommunicator.java

```diff
@@ -0,0 +1,24 @@
+package content;
+
+public class ContentCommunicator implements IContentCommunicator{
+    
+	public void initialiseUser(String userDetails) {
+    	//UI will call this to send any message to server
+		//this method will be implemented by content team.
+    }
+	
+	public void sendMessageToContent(String message) {
+    	//UI will call this to send any message to server
+		//this method will be implemented by content team.
+    }
+	
+	public void notifyUserExit() {
+    	//UI will call this to send any message to server
+		//this method will be implemented by content team.
+    }

```

Comment:
Follow the coding guidelines set for comments. Refer [this](https://google.github.io/styleguide/javaguide.html#s4.8.6-comments) for the comment format. In the comments write a short description of what the function does and what it returns(if any). Use @param to name and describe the parameters passed to the function.